EventStore/EventStore
# Manually trigger large Index Merges

Implemented new admin/mergeindexes http endpoint to trigger large Index Merge manually
Added a new AutomergeIndexes boolean option to deactivate the automatic index merges (default: true)

Fixes https://github.com/EventStore/EventStore/issues/1738

To test this PR 
1) set the new option AutomergeIndexes: False or --automerge-indexes=false
2) flood the test database with millions of test events
3) verify that the indexes are not automatically merged
4) make an http POST to the new endpoint: http://youreventstore:2113/admin/mergeindexes
5) verify that the indexes are merged
